doc/blog: atlas wilco quest announcement (#1356)
ariga/atlas: doc/website/blog/2022-12-20-wilco-atlas-quest.md

---
title: "Learning to solve migration issues in a realistic simulation game"
authors: rotemtam
tags: [simulation, learning, wilco]
image: https://atlasgo.io/uploads/images/explore_twitter_card.png
---

### Learning new things 

There are many ways to learn about a new topic. You can read books, watch videos, or even take a course. But one of the
most effective ways to learn is by doing. This is especially true when it comes to learning about a new tool or
technology. In order to truly understand how a tool works, you need to use it and see how it behaves in different
situations.

This is why I'm excited to announce that we've partnered with [Wilco](https://trywilco.com/) to create a new
kind of learning experience for Atlas users. Wilco develops a platform, built to emulate the conditions at a tech
startup, sends users on "quests" that cover everyday engineering tasks - from deploying an app to finding the root cause
of a production issue - utilizing real-life tech stacks.

When I first heard about Wilco, I was immediately intrigued by the idea of using a simulation game to

> teach people about database migrations and especially about solving all kinds of issues that many engineers

Reviewer: ```suggestion
teach people about database migrations. This platforms allows engineers to learn how to solve all kinds of issues that many 
```

Author: I will split into two but the stress here isn't on the platform but on the opportunity to practice "disastrous scenarios"

only hit during a serious outage when there's very little room for error.

### Introducing Wilco

When you use Wilco, you're put in the shoes of a software engineer at a fictional startup. You're given access
to a made up chat system humoristically called "Snack" where you meet different characters that you can interact with.

> You're also given access to a real GitHub repo and the means to run a real development environment on your local

Reviewer: ```suggestion
You are also given access to a real GitHub repo and the means to run a real development environment on your local
```

---
title: "Learning to solve migration issues in a realistic simulation game"
authors: rotemtam
tags: [simulation, learning, wilco]
image: https://atlasgo.io/uploads/images/explore_twitter_card.png
---

### Learning new things 

One of my favorite things about software engineering, is that it's a career of continuous learning.
Every new project or challenge presents us with an opportunity to increase our knowledge and improve our skills.

> Different people learn in different ways, but for me, one of the most effective ways to learn things has always been by

Reviewer: ```suggestion
Different people learn in different ways, but for me, one of the most effective ways to learn something new has always been by
```

> doing. Sure, reading books, watching video and immersing myself in technical documentation is foundational, but

Reviewer: ```suggestion
doing. Sure, reading books, watching videos and immersing myself in technical documentation is foundational, but
```

> I only feel like I really understand something is after I've applied it in some practical way.

Reviewer: ```suggestion
I only feel like I really understand something after I've applied it in some practical way.
```

This is why I'm excited to announce that we've partnered with [Wilco](https://trywilco.com/) to create a new
kind of learning experience for Atlas users. Wilco develops a platform, built to emulate the conditions at a tech
startup, sends users on "quests" that cover everyday engineering tasks - from deploying an app to finding the root cause
of a production issue - utilizing real-life tech stacks.

When I first heard about Wilco, I was immediately intrigued by the idea of using a simulation game to

> teach people about database migrations. Particularly interesting is the possibility to present people with

Reviewer: ```suggestion
teach people about database migrations. Particularly interesting, is the possibility to present people with
```
